sdk/storage/azure_storage_blob/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added support for SAS (Shared Access Signature) authentication scheme.
```

Comment:
Let's give a little more detail about what we actually changed. I think two entries, list the methods we added, and then a separate entry about SAS support via `from_blob_sas_url`.

Also from previous comment, mention breaking change on string type for container/blob name on the methods where we changed it.